src/plugins/console/server/lib/spec_definitions/json/generated/indices.simulate_index_template.json

```diff
@@ -0,0 +1,16 @@
+{
+  "indices.simulate_index_template": {
```

Comment:
This endpoint should also have its body included in the overrides:

https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-simulate-template.html#simulate-template-api-request-body

Reply:
Done

Reply:
Done